README.md

```diff
@@ -3,8 +3,98 @@
 [![Build Status](https://img.shields.io/travis/18F/glossary/master.svg)](https://travis-ci.org/18F/glossary)
 [![Test Coverage](https://img.shields.io/codecov/c/github/18F/glossary/master.svg)](https://codecov.io/github/18F/glossary)
 
-A work in progress.
+# Getting started
+# About
+Add a simple glossary panel to your site to help your users understand jargon-y terms. As seen on [betaFEC](https://beta.fec.gov) and [EITI](https://useiti.doi.gov/).
 
+## Download
+### Via npm (coming soon)
+```
+npm install @18f/glossary-panel
+```
+
+## Create your data
+Create a `terms.json` that follows the following pattern:
+
+```
+{
+  {
+    "term": "Glossary",
+    "definition": "A useful tool for finding the definitions of terms"
+  }
+}
```

Comment:
Take a look at the [`terms.json`](https://github.com/18F/glossary/blob/removing-classes/example/terms.json)...it's using that format. If the outermost thing is an Object (using curly braces), it needs a unique key for each Object within it, e.g.

```javascript
{
  "Glossary": {
    // ...
  },
  // ...
}
```

Does that make sense?